synapse/lib/stormtypes.py

```diff
@@ -3964,6 +3972,9 @@ async def iter(self):
         for item in items:
             yield item
 
+    async def set(self, prop, valu):
+        return await self.valu.set(prop, valu)
```

Comment:
You'll need to add perms checks against the layer here :) It's normally handled by the storm AST but methods are on their own :)

Reply:
( check out how node.data.set() does it, it'll be similar )

Reply:
Oh, and dont forget to add a test that confirms that it can explode with AuthDeny :D

Reply:
:+1: